pkg/util/util.go

```diff
@@ -45,7 +45,7 @@ func MaxTimeExceeded(startTime time.Time, maxWaitTime int64) bool {
 
 // Completed checks if the testrun is in a completed phase
 func Completed(phase argov1.NodePhase) bool {
-	if phase == argov1.NodeSucceeded || phase == argov1.NodeFailed || phase == argov1.NodeError || phase == argov1.NodeSkipped {
+	if phase == tmv1beta1.PhaseStatusSuccess || phase == tmv1beta1.PhaseStatusFailed || phase == tmv1beta1.PhaseStatusError || phase == tmv1beta1.PhaseStatusSkipped || phase == tmv1beta1.PhaseStatusTimeout {
```

Comment:
Optional/Idea: as discussed maybe it's good to introduce a list.contains utility function to make this line more compact and readable? (In combination with a dot import of tmv1beta1)
```suggestion
    if stringInSlice(phase, []string{PhaseStatusSuccess, PhaseStatusFailed, PhaseStatusError, PhaseStatusSkipped, PhaseStatusTimeout}) {
func stringInSlice(a string, list []string) bool {
    for _, b := range list {
        if b == a {
            return true
        }
    }
    return false
}
```

Reply:
Discussed offline